var/spack/repos/builtin/packages/minivite/package.py

```diff
@@ -62,4 +62,7 @@ def build_targets(self):
 
     def install(self, spec, prefix):
         mkdirp(prefix.bin)
-        install('miniVite', prefix.bin)
+        if (self.version == Version('1.0')):
+            install('dspl', prefix.bin)
+        elif (self.version == Version('1.1')):
```

Comment:
I would make that `>=`.

Reply:
Making it >= will always default to Version('1.0') install statement. They have changed the name of the target in the makefile, which results in an error if the checked out version is different.

Reply:
I'll just arrange it in descending order, so the latest version is installed first.

Reply:
Sorry, that is what I meant!